### PR TITLE
[ACTIVITI-3515] add liquibase to audit module/service

### PR DESCRIPTION
**Pull request by robfrank**
- add sql schema for h2, postgres and mysql
- update application properties to enable liquibase
- remove fabric8 maven plugin in favour of testcontainers to manage start and stop of rabbit and keycloak containers

### CODE REVIEW

File: activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/Application.java [hunk 1]
```diff
@@ -28,8 +31,25 @@
...
 public class Application {
 
+    static GenericContainer keycloakContainer = new GenericContainer("activiti/activiti-keycloak")
+            .withExposedPorts(8180)
+            .waitingFor(Wait.defaultWaitStrategy());
+
+    static RabbitMQContainer rabbitMQContainer = new RabbitMQContainer("rabbitmq:management");
+
+    static {
+        keycloakContainer.start();
```
Comment (bsideup) [inline activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/Application.java:41]: you may want to start both containers in parallel, e.g. with:
```
Startables.deepStart(Stream.of(keycloakContainer, rabbitMQContainer)).join();
```

Comment (robfrank) [reply activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/Application.java:41]: WOW, thanks for the suggestion

File: activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/EventsRestTemplate.java [hunk 2]
```diff
@@ -56,7 +56,7 @@ public EventsRestTemplate(ObjectMapper mapper, KeycloakTokenProducer keycloakTok
     }
 
     @Bean
-    public RestTemplateBuilder restTemplateBuilder(List<Module> modules) {
+    public RestTemplateBuilder restTemplateBuilder2(List<Module> modules) {
```
Comment (erdemedeiros) [inline activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/EventsRestTemplate.java]: I guess this was a temporary renaming, right?

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
Just one small detail. It looks good to me otherwise.
@robfrank also, some changes are coming to the database of this service, so it's probably better to wait for these changes before merging.

File: activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/EventsRestTemplate.java (hunk 2 above)
Comment (robfrank) [reply activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/EventsRestTemplate.java]: yes, due to problem in compilation, fixing

Comment (robfrank) [reply activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/EventsRestTemplate.java]: Maybe it is better to merge and handles the new change to the database with liquibase, adding a new changeset, WDYT?

**Review by erdemedeiros — APPROVED** [review]

**Comment by erdemedeiros** [comment]
Ref https://github.com/Activiti/Activiti/issues/2717

**Pull request merged**